.github/workflows/pr.yaml

```diff
@@ -132,7 +145,7 @@ jobs:
       build_type: pull-request
       script: ci/test_wheel_pylibraft.sh
   wheel-build-raft-dask:
-    needs: wheel-tests-pylibraft
+    needs: [checks, wheel-build-libraft]
```

Comment:
I suppose this is an intentional increasing of the parallelism of the build? I think we decided we liked starting the build job without waiting for prior test jobs in other repos so this seems fine, just making sure that we're aligning.